lib/app.js

```diff
@@ -283,6 +285,22 @@ function App () {
   app.post('/admin/federate', requireAdmin, bodyParser.urlencoded({ extended: true }), actions.admin.federate)
   app.post('/admin/retrieveFromWebOfRegistries', requireAdmin, actions.admin.retrieve)
 
+  app.get('/admin/sparql', requireAdmin, sparqlAdmin)
+  app.post('/admin/sparql', requireAdmin, bodyParser.urlencoded({ extended: true }), sparqlAdmin)
+
+  function sparqlAdmin (req, res) {
```

Comment:
Can this get moved down near the sparql function it is copied from?

Reply:
Sure.  I actually moved it up to be with the other admin pages though.  Which do you think is better?

Reply:
Wherever they are (I am partial to below all the routes, but this is a preference) they should be together, I think.

Reply:
Ah, you mean just the function?  Yes, I can move that.

Reply:
Yep, just the `sparqlAdmin` function.